CHANGES.rst

```diff
@@ -1,6 +1,19 @@
 Changes
 =======
 
+Version 0.6.0 (2019-12-19)
+--------------------------
+
+- Adds new API for Gitlab integration.
+- Adds new API for Kubernetes ingresses.
```

Comment:
For me, it sounds like it's REANA native API. What do you think about ``Adds new Kubernetes client API for ingresses``?

Reply:
Changed.